### PR TITLE
quast 3.2

### PR DESCRIPTION
**Pull request by bede**

### CODE REVIEW

**Comment by dpo** [comment]
The test should be executed in `testpath`, not `prefix`. Homebrew forbids write operations in the prefix.

**Comment by bede** [comment]
Done.
What's the best git workflow for revising a PR following critque? Additional commits or rebased commits? When you request I fix *existing* problems with formulae, should I commit those separately?

**Comment by bede** [comment]
@dpo @sjackman Any idea why the tests are failing?

**Comment by sjackman** [comment]
> What's the best git workflow for revising PRs following critique? Additional commits or rebased commits? When you request I fix existing problems with formulae, should I commit those separately?

Squashed and optionally rebased commits are preferred. Use `git commit --amend && git push --force`

**Comment by sjackman** [comment]
Compilation succeeded and bottles were built. `brew test quast` failed because it attempted to write to `…/libs/MUMmer3.23-osx/make.log`

http://bot.brew.sh/job/Homebrew%20Science%20Pull%20Requests/4129/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/test_homebrew_science_quast/
```
2016-02-08 03:49:24
Running Contig analyzer...
Compiling MUMmer (details are in /usr/local/Cellar/quast/3.2/libs/MUMmer3.23-osx/make.log and make.err)

[Errno 1] Operation not permitted: '/usr/local/Cellar/quast/3.2/libs/MUMmer3.23-osx/make.log'
```

Ideally this would be fixed, but if it's too much hassle, we can simply ignore it and merge. @dpo

File: quast.rb [hunk 1]
```diff
@@ -28,10 +28,8 @@ def install
   end
 
   test do
-    system "#{bin}/quast"
-    cd prefix do
+    cd testpath do
```
Comment (sjackman) [inline quast.rb]: I believe this is redundant. The `test` block is automatically run in `testpath`.

**Comment by bede** [comment]
Thanks @sjackman. Ran `audit` but not `test` on this one. Have revised test as advised. The test compiles a few packages which would presumably require involved tracking/patching? Do we need to fake the test or can you merge anyway?

Hadn't used `git commit --amend` before. Much simpler than rebasing, cheers.

**Comment by sjackman** [comment]
Yes, the first time that `quast` is run it compiles some software, which is a bit unusual. Ideally it would use the Homebrew version of that software, but that's a patch for another day. I'm okay with merging this patch as is.

**Comment by sjackman** [comment]
```
❯❯❯ brew pull --bottle https://github.com/Homebrew/homebrew-science/pull/3284
…
Error: Failure while executing: /usr/bin/curl -f#LA Homebrew\ 0.9.5\ (Ruby\ 2.0.0-645;\ OS\ X\ 10.11.3) --silent --fail -o /dev/null -I https://github.com/BrewTestBot/homebrew-science/compare/homebrew:master...pr-3284
```

**Comment by sjackman** [comment]
Perhaps the bottles timed out. I'll request a rebuild.

**Comment by sjackman** [comment]
I forgot to delete the old bottles from Bintray.
http://bot.brew.sh/job/Homebrew%20Bottles/43644/console
```
Error: quast-3.2.el_capitan.bottle.tar.gz is already published. Please remove it manually from
https://bintray.com/homebrew/bottles-science/quast/view#files
```
Building again.

**Pull request closed without merging**